server/api/crud/model_monitoring/deployment.py

```diff
@@ -837,7 +836,7 @@ def _create_monitoring_function_deletion_background_task(
             function_name,
             auth_info,
             background_task_name,
-            delete_v3io_stream,
+            delete_app_stream_resources,
```

Comment:
I would make `delete_monitoring_function` keyword argument only, as it has a lot of arguments and it's too easy to mess their order.